index.js

```diff
@@ -1,12 +1,6 @@
-wrapAsync = (Meteor.wrapAsync)? Meteor.wrapAsync : Meteor._wrapAsync;
+wrapAsync = Meteor.wrapAsync || Meteor._wrapAsync;
 Mongo.Collection.prototype.aggregate = function(pipelines) {
-  var coll;
-  if (this.rawCollection) {
-    // >= Meteor 1.0.4
-    coll = this.rawCollection();
-  } else {
-	// < Meteor 1.0.4
-    coll = this._getCollection();
-  }
-  return wrapAsync(coll.aggregate.bind(coll))(pipelines);
+  //         >= Meteor 1.0.4       < Meteor 1.0.4
+  var coll = this.rawCollection || this._getCollection;
```

Comment:
`this.rawCollection` and `this._getCollection` are methods that must be called. If you want to go the one-liner route, you could use the conditional ternary `var coll = this.rawCollection ? this.rawCollection() : this._getCollection();`.